addons/zeus/functions/fnc_contextualModules.sqf

```diff
@@ -0,0 +1,28 @@
+/*
+ * Author: SilentSpike
+ * Removes ace_zeus modules from zeus based on the prescence of other ACE addons
+ *
+ * Arguments:
+ * 0: The zeus logic <LOGIC>
+ * 1: The zeus player <UNIT>
+ *
+ * Return Value:
+ * nil
+ *
+ * Public: No
+ */
+
+private ["_logic","_addons","_removeAddons","_classname"];
+
+_logic = _this select 0;
+_addons = ["captives","medical"];
```

Comment:
This could probably be grabbed from the config instead of hard coded in sqf. 

New class:
```
class ACE_Curator {
    addons[] = {"captives", "medical"};
};
```

That way it's more in line with out framework design principles.